app/assets/javascripts/ng-app/controllers/address.js

```diff
@@ -54,14 +54,28 @@ angular.module('myApp')
     $scope.processing = false;
     $scope.error      = addressesService.error;
     $scope.results    = addressesService.results;  
-  };
+  }
 
   $scope.selectResult = function(result) {
     $scope.selectedResult = result;
-  }
+  };
```

Comment:
Remember to run a `$ git diff` prior to committing. This case it's nbd, but I've found it to be an awesome habit.